server/src/main/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinator.java

```diff
@@ -1348,6 +1353,21 @@ public int hashCode()
     }
   }
 
+  private static void bindColumnValuesToQueryWithInCondition(
```

Comment:
This looks identical to the private static method: `SqlSegmentsMetadataQuery#bindColumnValuesToQueryWithInCondition`. Can we reuse that method by making it package-private and removing this duplicate method?

Reply:
Reused, thanks.